css/index.html

```diff
@@ -98,7 +111,6 @@ <h2 class="title">Avatars</h2>
             </div>
         </section>
     </main>
-
 </body>
 
-</html>
+</html>
```

Comment:
Add newline to overcome this error
Reference: [https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log](url)